configure

```diff
@@ -676,32 +677,33 @@ for lib in $generated_META $lbytes; do
         -e "s|%%interfaces%%|${if}|g" \
         -e "s|%%findlib_version%%|${version}|g" \
         -e "s|%%natdynlink%%|${natdynlink}|g" \
-        -e "s|%%dynlink_dir%%|${dynlink_dir}|g" \
-        -e "s|%%unix_dir%%|${unix_dir}|g" \
-        -e "s|%%str_dir%%|${str_dir}|g" \
-        site-lib-src/$lib/META.in > site-lib-src/$lib/META
+        -e "s|%%dynlink_dir%%||g" \
+        -e "s|%%unix_dir%%||g" \
+        -e "s|%%str_dir%%||g" \
+        site-lib-src/"$lib"/META.in > site-lib-src/"$lib"/META
 
     echo "Configuration for $lib written to site-lib-src/$lib/META"
 done
 
-for part in `cd src; echo *`; do
-    if [ -f "src/$part/META.in" ]; then
-	sed -e "s/@VERSION@/$version/g" \
-            -e "s/@REQUIRES@/${req_bytes}/g" \
-            src/$part/META.in >src/$part/META
-    fi
-done
+tmp="$(mktemp)"
+find src ! -name "$(printf "*\n*")" -name 'META.in' -type f > "$tmp"
+while IFS= read -r part
+do
+    out=$(echo "$part" | sed -e "s/\.in\$//")
```

Comment:
`out=${part%.in}` perhaps?

Reply:
Indeed this is better. I had thought this was a bash-ishm but I tested with `dash` and `shellcheck` and it works!